tools/accuracy_checker/accuracy_checker/adapters/segmentation.py

```diff
@@ -109,6 +109,7 @@ def process(self, raw, identifiers=None, frame_meta=None):
         frame_meta = frame_meta or [] * len(identifiers)
         raw_outputs = self._extract_predictions(raw, frame_meta)
         for identifier, output in zip(identifiers, raw_outputs[self.output_blob]):
+            output = np.argmax(output, axis=0).reshape((1,)+output.shape[1:])
```

Comment:
it works for brain-tumor-segmentation-0001 only, for 0002 there should be another postprocessing

Reply:
Something like this:
prediction_.mask = (prediction_.mask > 0.5).astype(bool)
wt = prediction_.mask[0]
tc = prediction_.mask[1]
et = prediction_.mask[2]
pp = np.zeros(annotation_.mask.shape)
pp[wt] = 2
pp[tc] = 1
pp[et] = 3
prediction[0].mask = pp

Reply:
Ok, I'll add it to adapter, like `use_argmax`